README.md

```diff
@@ -9,3 +9,53 @@ This will get the terraform mixin and install it from source.
 1. `go get -u get.porter.sh/mixin/terraform`
 1. `cd $(go env GOPATH)/src/get.porter.sh/mixin/terraform`
 1. `make build install`
+
+
+## Examples
+
+### Install
+
+```yaml
+install:
+  - terraform:
+      description: "Install Azure Key Vault"
+      input: false
+      backendConfig:
+        key: "mybundle.tfstate"
+        storage_account_name: "mystorageacct"
+        container_name: "mycontainer"
+        access_key: "myaccesskey"
+      outputs:
+      - name: vault_uri
+```
+
+### Upgrade
```

Comment:
Can you add an invoke example? Since we don't have a general syntax like the other mixins, let's make sure we demonstrate each action.